Python/quadratic_roots.py

```diff
@@ -0,0 +1,21 @@
+# Solve the quadratic equation ax**2 + bx + c = 0
+
+import cmath
+
+a = 1
+b = 5
+c = 6
+
+# To take coefficients input from the users
+# a = float(input('Enter a: '))
+# b = float(input('Enter b: '))
+# c = float(input('Enter c: '))
+
+# calculate the discriminant
+d = (b**2) - (4*a*c)
+
+# find two solutions
+sol1 = (-b-cmath.sqrt(d))/(2*a)
+sol2 = (-b+cmath.sqrt(d))/(2*a)
+
+print('The solution are {0} and {1}'.format(sol1,sol2))
```

Comment:
It is good if you can make it function than just a plain program.

Reply:
I will make it into a function